frontend/src/EditTask.svelte

```diff
@@ -361,9 +362,27 @@ async function deleteTask(proceed) {
           {/if}
         </div>
 
-        <div title="All tasks with the same plagiarism key will be checked together">
-          <label for="plagiarism-key">Plagiarism key:</label>
-          <input type="text" id="plagiarism-key" maxlength="255" bind:value={task.plagiarism_key} />
+        <div
+          class="input-group mb-3"
+          title="All tasks with the same plagiarism key will be checked together">
+          <span class="input-group-text" id="basic-addon1">Plagiarism key:</span>
+          <input
+            class="form-control"
+            type="text"
+            id="plagiarism-key"
+            maxlength="255"
+            bind:value={task.plagiarism_key} />
+          <span class="input-group-text" id="basic-addon1">Task type:</span>
```

Comment:
The id here is duplicated with the one above. I think that we can remove the ids, also the `plagiarism-key` and `task_type` ids, they don't seem to be used anywhere.